Examples/php/value/runme.php

```diff
@@ -25,7 +23,8 @@
 
 	echo "\nNow I'm going to add the vectors together\n";
 
-        $r = new_vector();
+    $r = new Vector();
+	
```

Comment:
You've changed the indentation and added a blank line - please avoid whitespace changes mixed in with functional changes.  If the whitespace changes are needed, make them a separate commit.  Here they don't seem needed though.